server/libs/domain/test/fixtures.ts

```diff
@@ -0,0 +1,42 @@
+import { UserEntity } from '@app/infra';
+import { AuthUserDto } from '../src';
+
+export const ADMIN_AUTH_DTO = Object.freeze({
```

Comment:
This is a good idea to reuse some of these stubs. I think we should prepend the word mock (or even better `stub`) to these. e.g `adminAuthDtoStub`. I would prefer camelCase naming for these fixtures and mock/stub objects, but we can discuss it if you prefer SCREAMING_SNAKE_CASE (we have been using camelCase so far)

Reply:
I'm fine with adding stub/mock to the end, I usually use snake case when it's a constant and shouldn't change. I'm happy with either as long as it isn't `tooLongAndAnnoying`, otherwise it reminds me of Java and I get tilted :rofl:

Reply:
I'd also be fine with putting these in fixture json files or having objects here that nested so you access them via `stub.auth.admin` or something like that.

Reply:
I think TS objects would be better. With json you would have to "cast to the DTO type" everywhere is used (and it's easier to introduce a typo in a prop)